pypistats/cli.py

```diff
@@ -145,6 +164,7 @@ def overall(args):
     [
         argument("package"),
         argument("-v", "--version", help="eg. 2 or 3"),
+        arg_format,
```

Comment:
And here, we want:
```
        arg_format,
        arg_json,
```
(and not `arg_json` 5 lines below. Same for the other subcommands.)